test/integration/crcsuite/crcsuite.go

```diff
@@ -25,33 +25,149 @@ package crcsuite
 
 import (
 	"fmt"
+	"os"
+	"os/user"
+	"path/filepath"
 
 	"github.com/DATA-DOG/godog"
-	"github.com/code-ready/clicumber/testsuite"
+	"github.com/DATA-DOG/godog/gherkin"
+
+	clicumber "github.com/code-ready/clicumber/testsuite"
 )
 
-var sh testsuite.ShellInstance
+// FeatureContext defines godog.Suite steps for the test suite.
+func FeatureContext(s *godog.Suite) {
+
+	// CRC related steps
+	s.Step(`^removing CRC home directory succeeds$`,
+		CRCRemoveHome)
```

Comment:
Can we not use CRC in every step? This is unnecessary as all steps are related to CRC, as this is the Application Under Test.

Reply:
Good point. I'll update it. Although in this particular step we don't want to remove `$HOME` but `$HOME/.crc`, so I'd probably keep the CRC bit here. Or?

Reply:
@gbraad I removed 'CRC' from `CRCDelete` (it deletes CRC instance; used in e.g. BeforeFreature/AfterFeature) and from `CRCRemoveHome` (it removes `$HOME/.crc`; used in e.g. BeforeFeature)

I kept it in `CRCConfigFileContainsKey` - analogue of clicumber's `ConfigFileContainsKey` but for files placed inside `$HOME/.crc`. Same goes for `CRCFileExists` and `CRCConfigFileContainsKeyMatchingValue`.